pkg/builders/webhook/pod_webhook.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2024.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package webhook
+
+import (
+	admissionregistrationv1 "k8s.io/api/admissionregistration/v1"
+
+	kueue "github.com/openshift/kueue-operator/pkg/apis/kueueoperator/v1alpha1"
+)
+
+// These are webhooks that we only want to enable if the integration requests it
+// Otherwise we will remove them from the kueue manifest
+func dangerousWebhooks() []string {
```

Comment:
This could be a static array and not a function.

Reply:
Rename this to `optionalWebhooks` or something different. We do intend to deploy them on specific namespaces :)

Reply:
updated.

Reply:
Is there a reason why a function is preferred here? A global static array would produce less GC garbage.

Reply:
I wasn't sure how to declare a constant static array.

Reply:
updated.